src/infrastructure/torrent/cache_cleaner.go

```
@@ -44,22 +42,34 @@ func (m *manager) enforceCacheLimit() {
 		}
 		dir := filepath.Join(root, e.Name())
 		var size int64
-		filepath.WalkDir(dir, func(_ string, d fs.DirEntry, err error) error {
+		if err := filepath.WalkDir(dir, func(_ string, d fs.DirEntry, err error) error {
 			if err != nil {
 				return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 45) but it returns nil

(nilerr)